File: integrations/php/oicana-php/src/DiagnosticColor.php
<?php

declare(strict_types=1);

namespace Oicana;

/**
 * Color mode for compilation diagnostics.
 */
enum DiagnosticColor
{
    /** No colors in diagnostic output. */
    case None;
    /** ANSI codes for colors in diagnostic output. */
    case Ansi;
}

File: integrations/csharp/Oicana/Config/DiagnosticColor.cs
namespace Oicana.Config;

/// <summary>
/// Color mode for compilation diagnostics like warnings and errors.
/// </summary>
public enum DiagnosticColor
{
    /// <summary>
    /// No colors in diagnostic output.
    /// </summary>
    None = 0,

    /// <summary>
    /// ANSI codes for colors in diagnostic output.
    /// </summary>
    Ansi = 1,
}
